expression/builtin_string_test.go

```diff
@@ -984,6 +984,31 @@ func (s *testEvaluatorSuite) TestLpad(c *C) {
 	}
 }
 
+func (s *testEvaluatorSuite) TestQuote(c *C) {
+	defer testleak.AfterTest(c)()
+
+	tbl := []struct {
+		arg interface{}
+		ret interface{}
+	}{
+		{`Don\'t!`, `'Don\'t!'`},
+		{`Don't`, `'Don't'`},
```

Comment:
In MySQL:
mysql> select quote("Don't");
+----------------+
| quote("Don't") |
+----------------+
| 'Don\'t'       |
+----------------+
1 row in set (0.00 sec)